Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ConceptTypeEditController.java

```diff
@@ -108,9 +111,11 @@ public String editType(HttpServletRequest req, Principal principal,
 
         String userId = usersManager.findUser(principal.getName()).getUsername();
         String modified = type.getModified() != null ? type.getModified() : "";
-        if (!modified.trim().isEmpty())
-            modified += ", ";
-        type.setModified(modified + userId + "@" + (new Date()).toString());
+//        if (!modified.trim().isEmpty()) 
+//           modified+=",";
```

Comment:
remove if not needed

Reply:
actually, what has this been deleted? what about previous modifications? It's not pretty but at least we have it recorded.

Reply:
Even I'm not sure. I will check adhvik since he pushed this change and then I will update you.